modular_skyrat/modules/mapping/code/modules/genpop/turnstile.dm

```diff
@@ -61,7 +61,7 @@
 
 
 /obj/machinery/turnstile/Bumped(atom/movable/AM)
-	if(world.time - last_bumped > 5)
+	if(ismob(AM) && world.time - last_bumped > 5)
 		to_chat(usr, span_notice("[src] resists your efforts."))
```

Comment:
How the fuck do you have 47 commits for one change LMAO

Reply:
My smart ass decided to update from master instead of updating the branch. 🧠